dapp/src/utils/transformLinks.js

```diff
@@ -0,0 +1,33 @@
+const transformLinks = (links) => {
+  const sortOrder = (a, b) => {
+    return a.order - b.order
+  }
+
+  const linkedMapSet = links
+    .map((linkSet) => {
+      return {
+        href: linkSet.attributes.href,
+        label: linkSet.attributes.label,
```

Comment:
In some APIs, we don't return this nested `attributes` prop. Let's be consistent everywhere